gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/avatarpicker/AvatarPickerViewModel.kt

```diff
@@ -217,9 +230,9 @@ internal class AvatarPickerViewModel(
                     currentState.copy(
                         emailAvatars = emailAvatars,
                         scrollToIndex = if (scrollToSelected && emailAvatars.avatars.isNotEmpty()) {
-                            emailAvatars.avatars.indexOfFirst {
+                            emailAvatars.avatars.indexOfFirstOrNull {
```

Comment:
With the `coerceAtLeast(0)`, when there wasn't an avatar selected, we still updated the state with 0, so when we uploaded a new image and set it to 0 again, the `scrollToIndex` there wasn't a change in the state, and nothing happens. I think this is more correct and solves the issue.